tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -56,6 +60,12 @@ var (
 	bugReportDefaultIstioNamespace = "istio-system"
 	bugReportDefaultInclude        = []string{""}
 	bugReportDefaultExclude        = []string{strings.Join(sets.SortedList(inject.IgnoredNamespaces), ",")}
+
+	istioOperatorGVR = schema.GroupVersionResource{
```

Comment:
I think we've already have this GVR defined in some places.

Reply:
It was defined at multiple places previously as well. Moved it to a common place and re-using it now.